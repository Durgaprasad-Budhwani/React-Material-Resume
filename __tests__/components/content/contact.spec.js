/**
 * Created by durgaprasad on 10/13/17.
 */
import React from 'react';
import Contact from '../../../src/components/content/contact';
import renderer from 'react-test-renderer';

it('renders contact', () => {
  const tree = renderer.create(
    <Contact
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});