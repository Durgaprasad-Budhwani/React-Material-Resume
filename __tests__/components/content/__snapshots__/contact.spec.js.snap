// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders contact 1`] = `
<div
  className="MuiGrid-typeContainer-2 MuiGrid-typeItem-3 MuiGrid-spacing-xs-16-17 MuiGrid-justify-xs-space-around-15 About-container-1"
>
  <div
    className="MuiGrid-typeContainer-2 MuiGrid-typeItem-3 MuiGrid-spacing-xs-16-17 MuiGrid-justify-xs-center-12 MuiGrid-grid-xs-6-26"
  >
    <div
      className="MuiPaper-root-87 MuiPaper-shadow2-91 MuiPaper-rounded-88 ContactForm-container-85"
    >
      <form
        action="http://formspree.io/durgaprasad.budhwani@gmail.com"
        method="post"
      >
        <div
          className="MuiGrid-typeContainer-2 MuiGrid-direction-xs-column-4 MuiGrid-justify-xs-center-12"
        >
          <h2
            style={
              Object {
                "marginBottom": 0,
              }
            }
          >
            Feel free to contact me
          </h2>
          <div
            className="MuiFormControl-root-114 MuiFormControl-marginNormal-115"
            onBlur={[Function]}
            onFocus={[Function]}
            required={true}
          >
            <label
              className="MuiFormLabel-root-124 MuiInputLabel-root-118 MuiInputLabel-formControl-119 MuiInputLabel-animated-122"
              data-shrink={false}
              htmlFor="name"
            >
              NAME
              <span
                className=""
              >
                 *
              </span>
            </label>
            <div
              className="MuiInput-root-128 MuiInput-formControl-129 MuiInput-inkbar-130 MuiInput-underline-136"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              <input
                autoComplete={undefined}
                autoFocus={undefined}
                className="MuiInput-input-132 MuiInput-inputSingleline-139"
                defaultValue={undefined}
                disabled={false}
                id="name"
                name="name"
                onChange={[Function]}
                onKeyDown={undefined}
                onKeyUp={undefined}
                placeholder={undefined}
                readOnly={undefined}
                required={true}
                rows={undefined}
                type="text"
                value={undefined}
              />
            </div>
          </div>
          <div
            className="MuiFormControl-root-114 MuiFormControl-marginNormal-115"
            onBlur={[Function]}
            onFocus={[Function]}
            required={true}
          >
            <label
              className="MuiFormLabel-root-124 MuiInputLabel-root-118 MuiInputLabel-formControl-119 MuiInputLabel-animated-122"
              data-shrink={false}
              htmlFor="email"
            >
              EMAIL
              <span
                className=""
              >
                 *
              </span>
            </label>
            <div
              className="MuiInput-root-128 MuiInput-formControl-129 MuiInput-inkbar-130 MuiInput-underline-136"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              <input
                autoComplete={undefined}
                autoFocus={undefined}
                className="MuiInput-input-132 MuiInput-inputSingleline-139"
                defaultValue={undefined}
                disabled={false}
                id="email"
                name="email"
                onChange={[Function]}
                onKeyDown={undefined}
                onKeyUp={undefined}
                placeholder={undefined}
                readOnly={undefined}
                required={true}
                rows={undefined}
                type="email"
                value={undefined}
              />
            </div>
          </div>
          <div
            className="MuiFormControl-root-114 MuiFormControl-marginNormal-115"
            onBlur={[Function]}
            onFocus={[Function]}
            required={true}
          >
            <label
              className="MuiFormLabel-root-124 MuiInputLabel-root-118 MuiInputLabel-formControl-119 MuiInputLabel-animated-122"
              data-shrink={false}
              htmlFor="subject"
            >
              SUBJECT
              <span
                className=""
              >
                 *
              </span>
            </label>
            <div
              className="MuiInput-root-128 MuiInput-formControl-129 MuiInput-inkbar-130 MuiInput-underline-136"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              <input
                autoComplete={undefined}
                autoFocus={undefined}
                className="MuiInput-input-132 MuiInput-inputSingleline-139"
                defaultValue={undefined}
                disabled={false}
                id="subject"
                name="subject"
                onChange={[Function]}
                onKeyDown={undefined}
                onKeyUp={undefined}
                placeholder={undefined}
                readOnly={undefined}
                required={true}
                rows={undefined}
                type="text"
                value={undefined}
              />
            </div>
          </div>
          <div
            className="MuiFormControl-root-114 MuiFormControl-marginNormal-115"
            onBlur={[Function]}
            onFocus={[Function]}
            required={true}
          >
            <label
              className="MuiFormLabel-root-124 MuiInputLabel-root-118 MuiInputLabel-formControl-119 MuiInputLabel-animated-122"
              data-shrink={false}
              htmlFor="message"
            >
              MESSAGE
              <span
                className=""
              >
                 *
              </span>
            </label>
            <div
              className="MuiInput-root-128 MuiInput-formControl-129 MuiInput-inkbar-130 MuiInput-multiline-137 MuiInput-underline-136"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              <div
                className="MuiTextarea-root-143"
                style={
                  Object {
                    "height": 24,
                  }
                }
              >
                <textarea
                  aria-hidden="true"
                  className="MuiTextarea-shadow-145 MuiTextarea-textarea-144"
                  readOnly={true}
                  rows="1"
                  tabIndex={-1}
                  value=""
                />
                <textarea
                  aria-hidden="true"
                  className="MuiTextarea-shadow-145 MuiTextarea-textarea-144"
                  defaultValue={undefined}
                  readOnly={true}
                  rows={1}
                  tabIndex={-1}
                  value={undefined}
                />
                <textarea
                  autoComplete={undefined}
                  autoFocus={undefined}
                  className="MuiTextarea-textarea-144 MuiInput-input-132 MuiInput-inputMultiline-141"
                  defaultValue={undefined}
                  disabled={false}
                  id="message"
                  name="message"
                  onChange={[Function]}
                  onKeyDown={undefined}
                  onKeyUp={undefined}
                  placeholder={undefined}
                  readOnly={undefined}
                  required={true}
                  rows={1}
                  type="text"
                  value={undefined}
                />
              </div>
            </div>
          </div>
          <div>
            <button
              className="MuiButtonBase-root-160 MuiButton-root-146 MuiButton-raised-153 MuiButton-raisedPrimary-155 ContactForm-button-86"
              disabled={false}
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
              type="submit"
            >
              <span
                className="MuiButton-label-148"
              >
                SEND
              </span>
              <span
                className="MuiTouchRipple-root-162"
              />
            </button>
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="MuiGrid-typeContainer-2 MuiGrid-typeItem-3 MuiGrid-spacing-xs-16-17 MuiGrid-justify-xs-center-12 MuiGrid-grid-xs-6-26"
  >
    <div
      className="MuiPaper-root-87 MuiPaper-shadow2-91 MuiPaper-rounded-88 MapContainer-container-169"
    >
      <div
        className="MuiGrid-typeContainer-2 MuiGrid-direction-xs-column-4"
      >
        <div
          className="MapContainer-contactContainer-170"
          xs={4}
        >
          <div
            className="MuiGrid-typeContainer-2 MuiGrid-typeItem-3 MuiGrid-spacing-xs-24-18 MuiGrid-direction-xs-column-4"
          >
            <div
              className="MuiGrid-typeContainer-2 MuiGrid-typeItem-3 MuiGrid-spacing-xs-24-18 MuiGrid-align-xs-center-8 MuiGrid-justify-xs-center-12"
            >
              <div
                className="MuiGrid-typeItem-3 MuiGrid-grid-xs-3-23 ContactInfo-profileListItem-173"
              >
                Address:
              </div>
              <div
                className="MuiGrid-typeItem-3 MuiGrid-grid-xs-9-29 ContactInfo-profileListItemValue-174"
              >
                Pune, India
              </div>
            </div>
            <div
              className="MuiGrid-typeContainer-2 MuiGrid-typeItem-3 MuiGrid-spacing-xs-24-18 MuiGrid-align-xs-center-8 MuiGrid-justify-xs-center-12"
            >
              <div
                className="MuiGrid-typeItem-3 MuiGrid-grid-xs-3-23 ContactInfo-profileListItem-173"
              >
                Email:
              </div>
              <div
                className="MuiGrid-typeItem-3 MuiGrid-grid-xs-9-29 ContactInfo-profileListItemValue-174"
              >
                <a
                  className="ContactInfo-profileValueAItemValue-175"
                  href="mailto:durgaprasad.budhwani@gmail.com"
                >
                  durgaprasad.budhwani@gmail.com
                </a>
              </div>
            </div>
            <div
              className="MuiGrid-typeContainer-2 MuiGrid-typeItem-3 MuiGrid-spacing-xs-24-18 MuiGrid-align-xs-center-8 MuiGrid-justify-xs-center-12"
            >
              <div
                className="MuiGrid-typeItem-3 MuiGrid-grid-xs-3-23 ContactInfo-profileListItem-173"
              >
                Phone:
              </div>
              <div
                className="MuiGrid-typeItem-3 MuiGrid-grid-xs-9-29 ContactInfo-profileListItemValue-174"
              >
                <a
                  className="ContactInfo-profileValueAItemValue-175"
                  href="tel:+91-9545251359"
                >
                  +91-9545251359
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MapContainer-mapContainer-171"
          xs={8}
        />
      </div>
    </div>
  </div>
</div>
`;
